hack/generate-kustomize-patch-workload-identity-signing-key.sh

```diff
@@ -0,0 +1,11 @@
+# SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
Rename the file to `hack/generate-workload-identity-signing-key.sh`. The other scripts generate patch files while ours generates only a signing key. So it is wrong to have `generate-kustomize-patch` in the script name.